src/browser/atom-portable.coffee

```diff
@@ -0,0 +1,15 @@
+fs = require 'fs-plus'
+path = require 'path'
+
+module.exports =
+class AtomPortable
+  @portableAtomHomePath: ->
+    execDirectoryPath = path.dirname(process.execPath)
+    return path.join(execDirectoryPath, "../.atom")
```

Comment:
I think we usually write this as `path.join(execDirectoryPath, '..', '.atom')` so that the `/` isn't hard-coded